src/products/queries.ts

```diff
@@ -250,14 +249,54 @@ export const productMediaQuery = gql`
 `;
 
 export const gridAttributes = gql`
-  query GridAttributes($ids: [ID!]!) {
-    grid: attributes(first: 25, filter: { ids: $ids }) {
+  query GridAttributes($ids: [ID!]!, $hasAttributes: Boolean!) {
+    left: attributes(first: 10) {
```

Comment:
suggestion: using `left | right` doesn't mean what this data is, also it's not good practice to suggest that this part of query will be use in left side of ui.